internal/cloud/realm/command_test.go

```diff
@@ -0,0 +1,35 @@
+package realm_test
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	u "github.com/10gen/realm-cli/internal/utils/test"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+func TestRealmCommands(t *testing.T) {
+	u.SkipUnlessRealmServerRunning(t)
+
+	t.Run("should fail without an auth client", func(t *testing.T) {
+		client := realm.NewClient(u.RealmServerURL())
+
+		_, err := client.ListClusters(primitive.NewObjectID().Hex(), primitive.NewObjectID().Hex())
+		assert.Equal(t, realm.ErrInvalidSession{}, err)
+	})
+
+	t.Run("with an active session", func(t *testing.T) {
+		client := newAuthClient(t)
+		groupID := u.CloudGroupID()
+
+		t.Run("should list clusters", func(t *testing.T) {
+			app, err := client.CreateApp(groupID, "commands-test", realm.AppMeta{})
+			assert.Nil(t, err)
+
+			_, err = client.ListClusters(groupID, app.ID)
```

Comment:
[Q] Is this okay? There could be any number of available clusters or none.

Reply:
if im understanding this correctly, we can probably tighten this up via env vars and recommending devs to setup their own test projects to mirror the setup we have for the test user here (similar to how we leverage `u.CloudGroupName()`)